app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
```

Comment:
- [ ] It is best practice with Rails to use snake_case naming convention instead of  CamelCase. eg: likes_counter instead of LikesCounter
- [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_counter method every time a new like has been saved to database, then make the update_likes_counter method private
```rb
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:likes_counter)
  end
```